packages/manager/modules/advices/src/advices/component.js

```diff
@@ -0,0 +1,71 @@
+import controller from './controller';
+import template from './template.html';
+
+/**
+ * Makes 2api call to /adives to get advices for a given service type and name.
+ * It used serviceType, serviceName and queryParams to construct the URl.
+ *
+ * serviceType: type of service, refer 2api /advices documentation for list of service types supported.
+ *
+ * ServiceName: name of service
+ *
+ * queryParams: map of query params that need to pass to /advices api
```

Comment:
It could be nice to have an example, what kind of key/values are expected ? 
```suggestion
 * queryParams: object of query params that need to pass to /advices api
```
Or do we expect an instance of https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Map

Reply:
no its javascript object